File: redsheet-portal/src/app/database-models/v-security-answer.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vSecurityAnswer {
    constructor(vSecurityAnswer?: vSecurityAnswer )  {
        let properties = [ "securityAnswer", "securityQuestionName", "userId", "userName",];
        for (let property of properties)
            if (vSecurityAnswer && vSecurityAnswer[property])
                this[property] = vSecurityAnswer[property];
    }
 
    @maxLength(50)
	securityAnswer : string =   undefined;
 
    @required()
    @maxLength(50)
	securityQuestionName : string =   undefined;
 
	userId : number =   0 ;
 
    @maxLength(50)
	userName : string =   undefined;


}

File: RedSheet.DbEntities/Models/vSecurityAnswer.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vSecurityAnswers", Schema = "dbo")]
	[Serializable]
	public partial class vSecurityAnswer  
	{
 
	[MaxLength(50)]
	public string SecurityAnswer { get; set; } 
 
	[Required] 
	[MaxLength(50)]
	public string SecurityQuestionName { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int UserId { get; set; } 
 
	[MaxLength(50)]
	public string UserName { get; set; } 

	public vSecurityAnswer()
	{
    }

    }
}

